src/coreclr/jit/emitarm64.cpp

```diff
@@ -15364,7 +15452,7 @@ void emitter::emitDispLowPredicateReg(regNumber reg, PredicateType ptype, bool a
 //------------------------------------------------------------------------
 // emitDispArrangement: Display a SIMD vector arrangement suffix
 //
-void emitter::emitDispArrangement(insOpts opt)
+void emitter::emitDispArrangement(insOpts opt, bool addComma /* = false */)
```

Comment:
I don't see any use of calling `emitDispArrangement()` with `addComma` set to true.

Reply:
It is used at https://github.com/dotnet/runtime/pull/94811/files#diff-2b2c8b9011607926410624d6f81613fad7b74c6e0516d578675a8b792998fe4fR17249

Reply:
Thanks. (But still can be removed if end up using PREDICATE_SIZED)